Libraries/LibGUI/Variant.h

```diff
@@ -281,6 +297,7 @@ class Variant {
         RawPoint as_point;
         RawSize as_size;
         RawRect as_rect;
+        RawStringView as_string_view;
```

Comment:
Let's not do this. All other forms of `GUI::Variant` own their contained value. This suddenly introduces a subtype of `Variant` whose internal value has completely external ownership. That's a huge degradation in API quality.

Reply:
Hmmm...that would mean copying the string if some model does not implement the `data_matches()` API.
I guess if you're fine with it, I'll let it be.